src/pages/webinars.astro

```diff
@@ -1,7 +1,194 @@
 ---
 import Layout from "../layouts/Layout.astro";
+import Event from "../components/webinars/Event.astro";
+import Day from "../components/webinars/Day.astro";
 ---
 
 <Layout title="Digeek 2024">
-  <main>webinars</main>
+  <main class="w-auto h-auto bg-[#121212] pb-[50vh] pt-[100px] z-0">
+    <div
+      class="flex flex-col items-center justify-center z-0 md:max-lg:flex-row md:max-lg:ml-20 md:max-lg:items-start lg:ml-40 xl:ml-40 lg:items-start xl:flex-row xl:space-x-2 lg:justify-start"
+    >
+      <Day name="Marzo 14">
+        <Event
```

Comment:
En vez de crear manualmente cada Event deberias hacer un .map() a un json similiar a react. Documentacion: https://docs.astro.build/en/basics/astro-components/#the-component-template